CorsixTH/Lua/hospitals/player_hospital.lua

```diff
@@ -339,19 +339,20 @@ end
 --! Once a month the advisor may warn about long queues.
 --! Rooms requiring a doctor occasionally trigger the generic message
 function PlayerHospital:warnForLongQueues()
-  local queue_rooms, total_queue = {}, 0
+  local long_queue_rooms, active_rooms, total_queue = {}, 0, 0
   for _, room in pairs(self.world.rooms) do
-    if #room.door.queue then
+    if room.is_active and #room.door.queue then
       total_queue = total_queue + #room.door.queue
-    end
-    if #room.door.queue > 7 then
-      queue_rooms[#queue_rooms + 1] = room
+      active_rooms = active_rooms + 1
+      if #room.door.queue > 7 then
+        long_queue_rooms[#long_queue_rooms + 1] = room
+      end
     end
   end
-  if #queue_rooms == 0 or total_queue == 0 then return end
+  if #long_queue_rooms == 0 or total_queue == 0 then return end
```

Comment:
Not your fault, but if `#long_queue_rooms == 0` fails (ie it's larger than `0`), then `total_queue == 0` is also not going to hold, since a 'long queue room' has at least 8 people in the queue.